lib/grunt/utils.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
       done();
       return;
     }
-    var p = spawn('node', ['node_modules/protractor/bin/webdriver-manager', 'update']);
+    var p = spawn('node', [require.resolve('protractor/bin/webdriver-manager'), 'update']);
```

Comment:
That would only resolve what's exported, not the binary.

Have you tried specifying the binary as `'./node_modules/.bin/webdriver-manager'` instead of `node` and using [cross-spawn](https://www.npmjs.com/package/cross-spawn) for Windows compatibility?